src/main/java/com/google/devtools/build/lib/packages/StarlarkSemanticsOptions.java

```diff
@@ -623,6 +637,8 @@ public StarlarkSemantics toSkylarkSemantics() {
         .incompatibleDisallowOldOctalNotation(incompatibleDisallowOldOctalNotation)
         .incompatibleDisallowOldStyleArgsAdd(incompatibleDisallowOldStyleArgsAdd)
         .incompatibleDisallowStructProviderSyntax(incompatibleDisallowStructProviderSyntax)
+        .incompatibleDisallowRuleExecutionPlatformConstraintsAllowed(
```

Comment:
When introducing new flags, you also need to update SkylarkSemanticsConsistencyTest.java

Reply:
Done.